Tests/ttLib/tables/otTables_test.py

```diff
@@ -687,6 +687,33 @@ def test_splitMarkBasePos():
     ]
 
 
+class ColrV1Test(unittest.TestCase):
+  def setUp(self):
+    self.font = FakeFont(['.notdef', 'meh'])
+
+  def test_traverseEmptyPaintColrLayersNeedsNoLayerList(self):
+    colr = parseXmlInto(self.font, otTables.COLR(),
+      '''
+      <Version value="1"/>
+      <BaseGlyphList>
+        <BaseGlyphPaintRecord index="0">
+          <BaseGlyph value="meh"/>
+          <Paint Format="1"><!-- PaintColrLayers -->
+            <NumLayers value="0"/>
```

Comment:
@khaledhosny @rsheeter hm, a `NumLayers=0` is very odd, as the whole PaintColrLayers is no-op basically. Did nanoemoji produce that? Maybe the input SVG was completely empty (e.g. a whitespace glyph). I think in this case, we don't want to emit anything, i.e. the BaseGlyphPaintRecord.Paint should be None, not a no-op PaintColrLayers with NumLayers=0.

Reply:
Yes, that indeed the case. The glyph in question is `space` glyph and the SVG produced by picosvg is:
```xml
<svg xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 130.0 1500.0" width="130.0" height="1500.0">
  <defs/>
</svg>
```